packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -170,12 +170,14 @@ export default class WalletService {
         onCurrentWalletUpdated(wallets, newId)
       })
 
-    AddressesUsedSubject.getSubject().subscribe(async () => {
-      const currentWallet = this.getCurrent()
-      if (currentWallet) {
-        broadcastCurrentAddresses(currentWallet.id)
-      }
-    })
+    AddressDbChangedSubject.getSubject()
+      .pipe(debounceTime(DEBOUNCE_TIME))
```

Comment:
I think this value can be larger than `1000`

Reply:
Will the interval of events be larger than 1000ms?

Reply:
Will the interval of events be larger than 1000ms?